jboss/container/openjdk/jdk/11/configure.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+# Configure module
+set -e
+
+# As of rhel 7.6, rh-maven35 pulls in jdk8, so we need to remove them
+
+if [ -n "$(yum list installed java-1.8.0-openjdk-devel |grep java-1.8.0-openjdk-devel)" ]; then
```

Comment:
if rpm -q --quiet java-1.8.0-openjdk-devel; then

would be faster and not require a network round-trip (`yum list...` might cause a metadata refresh)

Reply:
Thanks @jmtd, I'll update the check.  Yesterday I was thinking about skipping it entirely and just doing something like, ` yum remove ... || echo "Java 8 not installed"`.